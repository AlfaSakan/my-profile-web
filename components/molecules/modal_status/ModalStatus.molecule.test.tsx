import renderer from 'react-test-renderer';
import ModalStatus from './ModalStatus.molecule';

describe('ModalStatus component test', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<ModalStatus message="test message" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
