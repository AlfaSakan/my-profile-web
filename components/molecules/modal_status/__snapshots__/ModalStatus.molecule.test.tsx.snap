// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalStatus component test should render correctly 1`] = `
<div
  className="fixed h-full w-full bg-black-200 top-0 flex items-center justify-center"
>
  <div
    className="w-1/3 h-1/6 rounded-2xl flex flex-col overflow-hidden bg-white justify-center"
    onClick={[Function]}
  >
    <div
      className="flex items-center px-5"
    >
      <div>
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          style={
            Object {
              "fill": "rgb(239 68 68)",
              "fontSize": 54,
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
          />
        </svg>
      </div>
      <p
        className="text-3xl ml-4 font-bold text-red-500"
      >
        test message
      </p>
    </div>
  </div>
</div>
`;
